js/src/ui/SelectionList/selectionList.css

```diff
@@ -58,7 +79,3 @@
   filter: grayscale(10%);
   opacity: 0.75;
```

Comment:
Well it's somehow related to this PR since it removes the `tick`, thus makes non-selected accounts less obvious : that's why I'm suggesting an opacity of 0.5... After all there are no such things as disabled accounts, so I'm not sure if that's a real issue ?